.artifact/tree.json

```diff
@@ -50624,72 +50624,6 @@
     },
     "example_cartographer_outputs": {
       "viam-office-02-22-1": {
-        "image_map": {
```

Comment:
should there be any additions in this file?

Reply:
turns out no, part of this PR we moved the artifacts from `position_new` to `position`. While we were discussing I was confused because of the hashs not being renamed in `position`, but really that was just github sliding over the changes in `position_new`